lib/pages/profile_info/profile_info_view.dart

```diff
@@ -0,0 +1,247 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:linagora_design_flutter/avatar/round_avatar_style.dart';
+import 'package:linagora_design_flutter/extensions/string_extension.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+import 'package:fluffychat/pages/profile_info/profile_info.dart';
+import 'package:fluffychat/pages/profile_info/profile_info_style.dart';
+import 'package:fluffychat/widgets/matrix.dart';
+import 'package:fluffychat/widgets/mxc_image.dart';
+
+class ProfileInfoView extends StatelessWidget {
+  final ProfileInfoController controller;
+  const ProfileInfoView(
+    this.controller, {
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final user = controller.user;
+    final contact = controller.widget.contact;
+    return Scaffold(
+      appBar: AppBar(
+        title: Row(
+          children: [
+            IconButton(
+              onPressed: controller.widget.onBack,
+              icon: const Icon(Icons.close),
+            ),
+            Text(
+              L10n.of(context)!.contactInfo,
+              style: Theme.of(context).textTheme.titleLarge,
+            ),
+          ],
+        ),
+      ),
+      body: SingleChildScrollView(
+        child: Center(
+          child: ConstrainedBox(
+            constraints:
+                const BoxConstraints(maxWidth: ProfileInfoStyle.maxWidth),
+            child: Builder(
+              builder: (context) {
+                if (contact?.matrixId != null) {
+                  return FutureBuilder(
+                    future: Matrix.of(context).client.getProfileFromUserId(
+                          contact!.matrixId!,
+                          getFromRooms: false,
+                        ),
+                    builder: (context, snapshot) => _Information(
+                      avatarUri: snapshot.data?.avatarUrl,
+                      displayName:
+                          snapshot.data?.displayName ?? contact.displayName,
+                      matrixId: contact.matrixId,
+                      email: contact.email,
+                      phoneNumber: null,
+                    ),
+                  );
+                }
+                if (contact != null) {
+                  return _Information(
+                    displayName: contact.displayName,
+                    matrixId: contact.matrixId,
+                    email: contact.email,
+                    phoneNumber: null,
+                  );
+                }
+                return _Information(
+                  avatarUri: user?.avatarUrl,
+                  displayName: user?.calcDisplayname(),
+                  matrixId: user?.id,
+                );
+              },
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _Information extends StatelessWidget {
+  static const double avatarRatio = 3 / 4;
```

Comment:
the image display here it is not show the good image (compare to avatar)

<img width="407" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/6462404/54efdc96-24df-4209-8c75-c33e78d54424">

<img width="97" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/6462404/44981994-011a-464f-bb47-49afa0534348">

Reply:
so what is the ratio here ?
acutually it can't 1:1 it not same with design, and also ugly

Reply:
Can we base on the image info to display it? Like we do in bubble :)